site/en/workflows.md

```diff
@@ -478,17 +587,11 @@ need to segment into lines in an extra step.
     </tr>
   </thead>
   <tbody>
-    <tr data-processor="ocrd-tesserocr-segment-region">
-      <td>ocrd-tesserocr-segment-region</td>
-      <td><code>-P find_tables false</code></td>
-      <td>Recommended</td>
-      <td><code>ocrd-tesserocr-segment-region -I OCR-D-DEWARP-PAGE -O OCR-D-SEG-REG</code></td>
-    </tr>
-    <tr data-processor="ocrd-segment-repair">
-      <td>ocrd-segment-repair</td>
-      <td><code>-P plausibilize true</code></td>
-      <td>Only to be used after <code>ocrd-tesserocr-segment-region</code></td>
-      <td><code>ocrd-segment-repair -I OCR-D-SEG-REG -O OCR-D-SEG-REPAIR -P plausibilize true</code></td>
+    <tr data-processor="ocrd-tesserocr-segment">
+      <td>ocrd-tesserocr-segment</td>
+      <td><code>-P find_tables false -P shrink_polygon true -P segmentation_level region -P textequiv_level none</code></td>
```

Comment:
Note: only `recognize` exposes the `level` params. And since we have `shrink_polygons` now, I think we can safely recommend _both_ `ocrd-tesserocr-segment-region` and `ocrd-tesserocr-segment` now.
```suggestion
      <td><code>-P find_tables false -P shrink_polygons true</code></td>
```